lib/Doctrine/ODM/PHPCR/Mapping/Annotations/Binary.php

```diff
@@ -21,9 +21,13 @@
 
 use Doctrine\Common\Annotations\Annotation;
 
+@trigger_error('The @Binary annotation is deprecated, use @Field(type="binary") instead.', E_USER_DEPRECATED);
```

Comment:
do we need to trigger_error? this was a huge mess with symfony, when suddenly tests start to fail because of deprecations. if we do we should at least explain in the changelog how to use the symfony deprecation handle to have succeeding tests even with those deprecations.

otherwise i vote to not trigger deprecation errors for now.